pkg/collect/helm.go

```diff
@@ -45,6 +49,32 @@ type VersionInfo struct {
 	Values    map[string]interface{} `json:"values,omitempty"`
 }
 
+type configGetter struct {
+	restConfig *rest.Config
+}
+
+// ToDiscoveryClient implements genericclioptions.RESTClientGetter.
+func (c configGetter) ToDiscoveryClient() (discovery.CachedDiscoveryInterface, error) {
+	return nil, nil
+}
```

Comment:
I am unclear if this function is used by the helm client. were you able to confirm either way? it may be possible to implement it like so.

```suggestion
func (c configGetter) ToDiscoveryClient() (discovery.CachedDiscoveryInterface, error) {
	discoveryClient, err := discovery.NewDiscoveryClientForConfig(c.restConfig)
	if err != nil {
		return nil, err
	}
	cached := memory.NewMemCacheClient(discoveryClient)
	return cached, nil
}
```

Reply:
I’ll check tomorrow, but from what I observed, only the implemented method was used. Helm operations there simply listed known resources, so it didn’t require discovery. Committed anyway, thanks for pointing that out.